README.md

```diff
@@ -36,10 +36,19 @@ details.
 
 Although this project is not part of the [CNCF][cncf], we abide by its [code of conduct][cncf-conduct].
 
+## License
+
+This software is licensed under the [Apache License, Version 2.0][asl2].
+
+Additionally, the End User License Agreement included in the [`EULA.pdf`](./EULA.pdf) file applies to software artifacts
+released by TriggerMesh Inc.: compiled executables, container images, etc.
```

Comment:
Note: I wasn't sure about the "etc." here. Should we keep the list exhaustive instead, so it's not left to interpretation?

Reply:
I think we can remove the `etc`